tasks/prime_endpoint_workflows.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from locust import tag, task
+
+from .prime import PrimeTasks, SupportTasks
+from utils.constants import MOVE_TASK_ORDER
+
+logger = logging.getLogger(__name__)
+
+
+@tag("workflow")
```

Comment:
I might make this workflow tag different from the other one, just so it can also be used to differentiate from the other workflow. But, frankly, I don't know if it's necessary - we'll probably be using the tags on the tasks themselves to do that